### PR TITLE
chore: Add tracking to the version number for better insights on user stats

### PR DESCRIPTION
**Pull request by ifrost**
### ✨ Description

**Related issue(s):**

N/A

### 📖 Summary of the changes

Adding version to all tracked events to better distinguish usage stats

### 🧪 How to test?

Run following command in dev tools:

```
window._debug.echo.enable();
```

You should see the version being tracked as an extra property:

Screenshot 2024-09-04 at 15 23 06

### CODE REVIEW

**Review by bryanhuhta — APPROVED** [review]
LGTM

**Comment by ifrost** [comment]
Back to draft to check if we can get this info directly from runtime

File: src/app/App.tsx [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
 
 export function App() {
+  const {
+    meta: {
+      info: { version },
+    },
+  } = usePluginContext();
+
+  useEffect(() => {
+    setTrackingVersion(version);
```
[the remark this answers is not in the document]
Comment (ifrost) [reply src/app/App.tsx]: Updated in c95aa9f65748df45158c563120d722d6dbd17c0e

**Review by grafakus — APPROVED** [review]
LGTM, thx!

**Pull request merged**